packages/amplify-util-mock/src/__tests__/velocity/relational-auth.test.ts

```diff
@@ -3,11 +3,22 @@ import { ModelTransformer } from '@aws-amplify/graphql-model-transformer';
 import { PrimaryKeyTransformer, IndexTransformer } from '@aws-amplify/graphql-index-transformer';
 import { HasManyTransformer, HasOneTransformer, BelongsToTransformer } from '@aws-amplify/graphql-relational-transformer';
 import { GraphQLTransform } from '@aws-amplify/graphql-transformer-core';
-import { AppSyncAuthConfiguration } from '@aws-amplify/graphql-transformer-interfaces';
```

Comment:
seems like the diff in this file may be due to not rebasing?

Reply:
I'll check, I did have to apply one commit that wasn't in the script to get this working, since [this commit](https://github.com/aws-amplify/amplify-cli/commit/fd2e4ff706ea3dccc609762a295bbeec3da20bd5) introduced a dep on an internal test file, which can't be accessed via the npm dependency.

Reply:
Yep, just confirming this is part of the one commit I had to manually apply to fix that test, commit is here https://github.com/aws-amplify/amplify-cli/pull/10323/commits/518d4c0e8f93dc68a5335b2a1ab50b2448843c3e

Reply:
Ah gotcha. Okay LGTM!